### ISSUE TITLE
Finish migration of `stage0-posix` to Nixpkgs

### ISSUE THREAD

**Issue by fricklerhandwerk** [ngi-nix/ngipkgs#304]
### Repository URL

https://github.com/ngi-nix/mes/tree/master/stage0

### Notes

All the tools are already in Nixpkgs: https://github.com/NixOS/nixpkgs/pull/227914/files

But one can't find anything for `mes`, `hex0` and the other binaries on search.nixos.org: https://search.nixos.org/packages?show=make-minimal-bootstrap-sources

Make it such that searching for executables will produce the package name.

**Comment by A-jay98** [comment]
Is this related?

https://github.com/NixOS/nixpkgs/blob/db4b034aef336492d25f8be92d2a12cbdec0695f/pkgs/top-level/packages-config.nix#L38-L42

**Comment by fricklerhandwerk** [comment]
No, the comment is only about intermediate tools required for bootstrapping, which may be out of date, such as years-old Bash or GCC.

**Comment by GetPsyched** [comment]
Considering we've run out of time, this seems to be, to the best of our efforts, possibly the majority of the progress toward closing this issue.

https://github.com/NixOS/nixpkgs/pull/322422
